components/server/ee/src/prebuilds/prebuilt-status-maintainer.ts

```diff
@@ -100,6 +100,8 @@ export class PrebuildStatusMaintainer implements Disposable {
     protected getConclusionFromPrebuildState(pws: PrebuiltWorkspace): "error" | "failure" | "pending" | "success" {
         if (pws.state === "aborted") {
             return "error";
+        } else if (pws.state === "queued") {
```

Comment:
TIL this is used for reporting status back to GitHub only, and has effectively no impact (besides being easier on logging), so: :+1: